src/lib/viewers/doc/DocPreloader.js

```diff
@@ -60,10 +64,13 @@ class DocPreloader extends EventEmitter {
      * [constructor]
      *
      * @param {PreviewUI} previewUI - UI instance
+     * @param {Object} options - Preloader options
+     * @param {Api} options.api - API Instance
      * @return {DocPreloader} DocPreloader instance
      */
-    constructor(previewUI) {
+    constructor(previewUI, { api } = {}) {
```

Comment:
It looks like this is being passed in as a top-level param rather than an options object.